src/modal-template.ts

```diff
@@ -71,7 +71,15 @@ export class ModalTemplate extends LitElement {
    * @private
    * @memberof ModalTemplate
    */
-  private handleCloseButton(): void {
+  private handleCloseButton(e: Event): void {
+    e.preventDefault();
+    if (
+      e.type === 'keydown' &&
+      (e as KeyboardEvent).key !== ' ' &&
+      (e as KeyboardEvent).key !== 'Enter'
+    ) {
+      return;
```

Comment:
Just noting the codecov warning that we seem to be missing a test here.

Reply:
I'll take a look.

Reply:
cleaned up